My story into blogs (#2563)
DFE-Digital/get-into-teaching-app: app/views/content/blog/returning-to-teaching-with-interational-experience.md

### Moving home

Moving from China to England was very easy. Like most international schools, mine had a comprehensive repatriation package and arranged my flights and shipment. They even arranged for my cat to be repatriated for me!

### Back in an English school

Starting back at an English state school was hard, like starting any new job - so much to learn! But it didn't feel different just because I was back in England. By October half term, my new school felt like home.

Though I was only away for 2 years, during that time a new GCSE curriculum and Ofsted Framework were introduced!

While in China, I'd kept up-to-date with the UK education scene via Twitter. I visited UK schools and went on UK-based courses whenever I was home in England for the holidays.

As for behaviour management – teenagers are teenagers! I still had to manage behaviour in China in the same way I did back in London, so I didn't lose any of my skills there!

I'm enjoying the freedom to teach the way I want to. I felt a huge amount of pressure in a fee-paying international school (where parents call a certain amount of the shots!) to teach in a certain way.

> It's so much easier here to engage with research, learn new things and try them out in the classroom - without being judged! I feel professionally free in England.

Reviewer: This bit is weirdly coming up as indented for me?

---
title: Returning to teaching with international experience
date: "2019-10-07"
author: Katie Lockett
images:
  q-katie:
    path: "media/images/content/blog/stories-katie.jpg"
    thumbnail_path: "media/images/content/blog/stories-katie.jpg"
description: |-
  Head of faculty Katie Lockett shares her story of returning to teach in England from China.
keywords:
  - life as a teacher
  - day in the life
  - classroom
tags:
  - non-UK teachers
---

$q-katie$

> After four years of teaching modern foreign languages in London, I decided to try something different. I moved to China to teach in an international school, but after a while, I wanted to move back to England to be closer to my partner.

Reviewer: ```suggestion
After 4 years of teaching modern foreign languages in London, I decided to try something different. I moved to China to teach in an international school, but after a while, I wanted to move back to England to be closer to my partner.
```

> After four years of teaching modern foreign languages in London, I decided to try something different. I moved to China to teach in an international school, but after a while, I wanted to move back to England to be closer to my partner.

Reviewer: ```suggestion
After 4 years of teaching modern foreign languages in London, I decided to try something different. I moved to China to teach in an international school, but after a while, I wanted to move back to England to be closer to my partner.
```

---
title: Returning to teaching with international experience
date: "2019-10-07"
author: Katie Lockett
images:
  q-katie:
    path: "media/images/content/blog/stories-katie.jpg"
    thumbnail_path: "media/images/content/blog/stories-katie.jpg"
description: |-
  Head of faculty Katie Lockett shares her story of returning to teach in England from China.
keywords:
  - life as a teacher
  - day in the life
  - classroom
tags:

>   - non-UK teachers

Reviewer 2: I think we should add another tag as well @EllieNodder 
* returning to teaching

---

$q-katie$

After four years of teaching modern foreign languages in London, I decided to try something different. I moved to China to teach in an international school, but after a while, I wanted to move back to England to be closer to my partner.

We're fortunate to have so many research-based educators here who share their ideas at exciting events like ResearchEd.

I missed being at the cutting edge of educational research, like I felt I had been while working in London.

I started looking for jobs in January. I looked in the normal places like ETeach and TES.

## Interviews from a distance

> Before I applied, I always emailed the head teacher of the school to ask if they would consider international applicants and if they would interview me on Skype.

Reviewer 2: ```suggestion
Before I applied, I always emailed the headteacher of the school to ask if they would consider international applicants and if they would interview me on Skype.
```

They were always grateful to me for being proactive and considerate of the fact that interviewing by Skype was a new concept to them. I applied for about 10 jobs and was offered 4 Skype interviews.

> I was offered a job in February, 2 months after I started looking. I was used to Skype interviews from applying to international schools, so I probably found it less daunting than the head teachers did!

Reviewer 2: ```suggestion
I was offered a job in February, 2 months after I started looking. I was used to Skype interviews from applying to international schools, so I probably found it less daunting than the headteachers did!
```

My international school was very supportive. They contacted the head teachers in England to describe my teaching. They also provided permission for me to share performance management reviews and videos of me teaching.

## Moving home

Moving from China to England was very easy. Like most international schools, mine had a comprehensive repatriation package and arranged my flights and shipment. They even arranged for my cat to be repatriated for me!

## Back in an English school

Starting back at an English state school was hard, like starting any new job - so much to learn! But it didn't feel different just because I was back in England. By October half term, my new school felt like home.

> Though I was only away for 2 years, during that time a new GCSE curriculum and Ofsted Framework were introduced!

Reviewer 2: Should framework be lowercase? And should we define Ofsted?

## Back in an English school

Starting back at an English state school was hard, like starting any new job - so much to learn! But it didn't feel different just because I was back in England. By October half term, my new school felt like home.

Though I was only away for 2 years, during that time a new General Certificate of Secondary Education (GCSE) curriculum and Ofsted framework[^1] were introduced.

While in China, I'd kept up-to-date with the UK education scene via Twitter. I visited UK schools and went on UK-based courses whenever I was home in England for the holidays.

As for behaviour management – teenagers are teenagers. I still had to manage behaviour in China in the same way I did back in London, so I didn't lose any of my skills there.

I'm enjoying the freedom to teach the way I want to. I felt a huge amount of pressure in a fee-paying international school (where parents call a certain amount of the shots) to teach in a certain way.

It's so much easier here to engage with research, learn new things and try them out in the classroom - without being judged. I feel professionally free in England.

> ### Advice for teachers planning to return to England

Reviewer 2: ```suggestion
## Advice for teachers planning to return to England
```
